### ISSUE TITLE
[NextJS]: Content block - Mixed groups cannot tab to link list

### ISSUE THREAD

**Issue by RichKummer** [carbon-design-system/carbon-for-ibm-dotcom#8314]
### Description

On Firefox with VoiceOver, I cannot tab to the `link list` items inside `Content block - mixed`. I tab past the cards and then immediately move to the next component, but I should tab up to the link list items before moving on to the next component.

https://user-images.githubusercontent.com/15643582/154334772-ba697fde-308b-4a13-a7b3-5a50ad684174.mov

### Component(s) impacted

Content block - mixed items

### Browser

Firefox

### Carbon for IBM.com version

v1.31.0-rc.1

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

C4IBM.com

### Package

@carbon/ibmdotcom-react

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom-nextjs-test/next/example-page-a.html

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
